lib/src/command/login.dart

```diff
@@ -40,7 +73,7 @@ class LoginCommand extends PubCommand {
     }
   }
 
-  Future<_UserInfo> retrieveUserInfo() async {
+  Future<_UserInfo> _retrieveUserInfo() async {
```

Comment:
> Are you just making this private because it's not used anywhere else?

Actually I saw other methods are private so I thought you might forget to make this one too.

Reply:
Done ✅

https://github.com/dart-lang/pub/pull/3010